public/views/content/studentData.html

```diff
@@ -4,7 +4,7 @@
     <!-- Reading level -->
     <center><b>{{thisMonth | date: 'MMMM'}}</b></center>
     <br>
-    <span tooltip-placement="left" uib-tooltip="Current reading level" class="glyphicon glyphicon-book" aria-hidden="true"></span> Lvl X
+    <span tooltip-placement="left" uib-tooltip="Current reading level" class="glyphicon glyphicon-book" aria-hidden="true"></span> Lvl {{currentStudent.reading}}
```

Comment:
Oh. I have no idea. I think this line had a merge conflict and I wasn't sure which one was the correct one to choose, I guess I chose the wrong one? Feel free to change it back